app/src/main/java/com/courseproject/tindar/usecases/likelist/LikeListDsGateway.java

```diff
@@ -0,0 +1,14 @@
+package com.courseproject.tindar.usecases.likelist;
+
+public interface LikeListDsGateway {
```

Comment:
class javadoc

Reply:
This documentation I can take or leave, but we might as well include it while we're at it.